Detect.py

```diff
@@ -7,25 +7,22 @@
 # You may need to install requests and uuid.
 # Run: pip install requests uuid
 
-
 import os, requests, uuid, json
 
-# Checks to see if the Translator Text subscription key is available
-# as an environment variable. If you are setting your subscription key as a
-# string, then comment these lines out.
-if 'TRANSLATOR_TEXT_KEY' in os.environ:
-    subscriptionKey = os.environ['TRANSLATOR_TEXT_KEY']
-else:
-    print('Environment variable for TRANSLATOR_TEXT_KEY is not set.')
-    exit()
-# If you want to set your subscription key as a string, uncomment the next line.
-#subscriptionKey = 'put_your_key_here'
+key_var_name = 'TRANSLATOR_TEXT_SUBSCRIPTION_KEY'
+if not key_var_name in os.environ:
+    raise Exception('Please set/export the environment variable: {}'.format(key_var_name))
+subscriptionKey = os.environ[key_var_name]
```

Comment:
Hey Jason, can we adjust this to `subscription_key` to match python standards. This was an error in my old samples as well. This is true for all samples.

Reply:
Done.